### ISSUE TITLE
Access more Image member functions

### ISSUE THREAD

**Issue by jim-easterbrook** [LeoHsiao1/pyexiv2#84]
Would it be possible to add some more of the Image class member functions to the Python interface? I'd like to be able to use `Exiv2::Image::checkMode` (https://www.exiv2.org/doc/classExiv2_1_1Image.html#a9044deeee647a5c646d90d4285ecde66) and `Exiv2::Image::mimeType` (https://www.exiv2.org/doc/classExiv2_1_1Image.html#a5c517b500586613595e2e1b62601657a).

**Comment by jim-easterbrook** [comment]
Another thing I'd like to access is `Exiv2::XmpProperties::registerNs` (https://www.exiv2.org/doc/classExiv2_1_1XmpProperties.html#ae58ee081625b7924563e93a1ba184fec)

**Comment by LeoHsiao1** [comment]
So far, I've mapped only a few Exiv2 methods because they're familiar. Of course other methods can also be mapped.
I'm traveling this weekend, so I'll try this next weekend.

**Comment by LeoHsiao1** [comment]
I have mapped `Exiv2::Image::mimeType` and `Exiv2::Image::checkMode`. The sample:
```py
>>> import pyexiv2
>>> img = pyexiv2.Image(r'./pyexiv2/tests/data/1.jpg')
>>> img.get_mime_type()
'image/jpeg'
>>> img.get_access_mode()
{'exif': 'read+write', 'iptc': 'read+write', 'xmp': 'read+write', 'comment': 'read+write', 'icc': None}
```
But Exiv2 does not seem to check the access mode for ICC profile:
https://github.com/Exiv2/exiv2/blob/c9f253f0e5674194045db8f78e53ce7e647b8a69/src/image.cpp#L803

So I'm going to hide the access mode for ICC profile. The sample:
```py
>>> img.get_access_mode()
{'exif': 'read+write', 'iptc': 'read+write', 'xmp': 'read+write', 'comment': 'read+write'}
```

**Comment by LeoHsiao1** [comment]
I have mapped `Exiv2::XmpProperties::registerNs` . The sample:
```py
>>> img.modify_xmp({'Xmp.test.mytag1': 'Hello'})
RuntimeError: No namespace info available for XMP prefix `test'
>>> pyexiv2.registerNs('a namespace for test', 'Ns1')
>>> img.modify_xmp({'Xmp.Ns1.mytag1': 'Hello'})
>>> img.read_xmp()['Xmp.Ns1.mytag1']
'Hello'
```
I will release the new version tomorrow.

**Comment by LeoHsiao1** [comment]
I have released [v2.7.0](https://github.com/LeoHsiao1/pyexiv2/releases/tag/v2.7.0) to GitHub and pypi.org .
